src/EvLJson/Parser.go

```diff
@@ -240,23 +352,33 @@ func handleExponentCoefficientNegative(p *Parser, b byte) signal_t {
 func handleExponentCoefficientLeadingZero(p *Parser, b byte) signal_t {
 	if b >= '1' && b <= '9' {
 		p.handle = handleExponentCoefficientEnd
-		return SIG_NEXT_BYTE
+		return signalDataNextByte(p, b)
 	}
 	if b == '0' {
-		return SIG_NEXT_BYTE
+		return signalDataNextByte(p, b)
 	}
-	// TODO: exponent only had /0+/ for the exponent
-	// signal this if it is important
 	popHandle(p)
-	return SIG_REUSE_BYTE
+	return yieldToUserSig(p, SIG_REUSE_BYTE)
+}
+
+func handleExponentCoefficientStricterLeadingZero(p *Parser, b byte) signal_t {
+	if b >= '1' && b <= '9' {
+		p.handle = handleExponentCoefficientEnd
+		return signalDataNextByte(p, b)
+	}
+	if b == '0' {
+		return signalDataNextByte(p, b)
```

Comment:
Logic issue here. Should return an error signal and set the err member to a new error type around invalid stricter exponent format.